homeassistant/core.py

```diff
@@ -99,9 +100,10 @@
     from .config_entries import ConfigEntries
 
 
-STAGE_1_SHUTDOWN_TIMEOUT = 100
-STAGE_2_SHUTDOWN_TIMEOUT = 60
-STAGE_3_SHUTDOWN_TIMEOUT = 30
+AUTOMATION_STAGE_SHUTDOWN_TIMEOUT = 20
```

Comment:
Lots call this stop pending to match the event name since something else might want to use that